src/shared/scrapers/USA/CA/sonoma-county.js

```diff
@@ -14,6 +15,9 @@ const scraper = {
   async scraper() {
     const $ = await fetch.page(this.url);
     const $th = $('th:contains("Total in Sonoma County")');
+    if ($th.html() === null) {
+      throw new DeprecatedError('Could not find table');
```

Comment:
So I don't think not finding the table means we call the source deprecated -- we still want the error. The fact is, `parse.number` should throw and the thing should fail, and we should get that error....